liquibase-core/src/main/java/liquibase/diff/output/changelog/DiffToChangeLog.java

```diff
@@ -608,11 +608,9 @@ protected void addDependencies(DependencyUtil.DependencyGraph<String> graph, Lis
                 String tabName = StringUtil.trimToEmpty((String)row.get("REFERENCED_SCHEMA_NAME")) +
                         "." + StringUtil.trimToEmpty((String)row.get("REFERENCED_NAME"));
 
-                if (!(tabName.isEmpty() || bName.isEmpty())) {
```

Comment:
Why this if block is removed? Is there not chance both `tabName` and `bName` are empty?

Reply:
Because Condition '(fieldValue instanceof DatabaseObject) && !snapshotControl.shouldInclude(((DatabaseObject) ...' is always 'false'